draft-ietf-ice-rfc5245bis.xml

```diff
@@ -1142,8 +1068,9 @@ reflexive candidate.
 <section anchor="sec-prioritizing" title="Prioritizing Candidates">
 <t>
 The prioritization process results in the assignment of a priority to
-each candidate. Each candidate for a media stream MUST have a unique
-priority that MUST be a positive integer between 1 and (2**31 - 1).
+each candidate. Each candidate MUST have a unique
+priority (unique in the scope of a component of a media stream in a session)
```

Comment:
I think old text was correct. Should be unique across components and the formula takes care of that.